### PR TITLE
Notif Setting Page

### PR DESCRIPTION
**Pull request by Mussuranae**
Rework notification setting page

### CODE REVIEW

File: libs/user/src/lib/auth/forms/notifications-form/notifications-form.component.ts [hunk 1]
```diff
@@ -14,11 +17,148 @@ import { NotificationType } from '@blockframes/notification/types';
...
+      },
+      subNotification: [
+        {
+          notificationType: 'requestFromUserToJoinOrgCreate',
+          subtitle: 'A new user requests to join your organization.',
+          email: true,
+          app: true
+        },
+        {
+          notificationType: 'memberAddedToOrg',
```
Comment (phpgeekfr) [inline libs/user/src/lib/auth/forms/notifications-form/notifications-form.component.ts:36]: replace by orgMemberUpdated

File: libs/user/src/lib/auth/forms/notifications-form/notifications-form.component.ts [hunk 2]
```diff
@@ -14,11 +17,148 @@ import { NotificationType } from '@blockframes/notification/types';
...
+        email: true,
+        app: true
+      },
+      subNotification: [
+        // {
+        //   notificationType: '',
+        //   subtitle: 'An organization invites you to a screening.',
+        //   email: true,
+        //   app: true
+        // },
```
Comment (phpgeekfr) [inline libs/user/src/lib/auth/forms/notifications-form/notifications-form.component.ts:79]: please add existing notification in develop  CF libs/user/src/lib/auth/forms/notifications-form/notifications-form.component.html

File: libs/user/src/lib/auth/forms/notifications-form/notifications-form.component.ts [hunk 3]
```diff
@@ -14,11 +17,148 @@ import { NotificationType } from '@blockframes/notification/types';
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class NotificationsFormComponent {
+  public notifications = [
```
Comment (phpgeekfr) [inline libs/user/src/lib/auth/forms/notifications-form/notifications-form.component.ts:20]: I would prefere `  public notifications = [` => `  public notificationsGroups = [`
`      subNotification: [` => `notification: [`

**Review by phpgeekfr — APPROVED** [review]

File: libs/user/src/lib/auth/forms/notifications-form/notifications-form.component.ts (hunk 2 above)
Comment (Mussuranae) [reply libs/user/src/lib/auth/forms/notifications-form/notifications-form.component.ts]: Done

**Pull request merged**